benches/payout-stakers/src/main.rs

```diff
@@ -1,4 +1,5 @@
 use aleph_client::create_connection;
+use aleph_client::staking_bond;
```

Comment:
Could you please collapse `use`s

Reply:
Maybe helpful, with this in fmt config:
```
edition = "2021"
imports_granularity = "Crate"
use_field_init_shorthand = true
reorder_imports = true
reorder_modules = true

```
I was able to add rust fmt to the on-save hook (in emacs, should be possible with other IDEs too)

Reply:
done, but had to do it manually